### ISSUE TITLE
Make --no-total remove all totals [$5]

### ISSUE THREAD

**Issue by makdisse** [simonmichael/hledger#1234]
Example: https://hastebin.com/kaqopenago.rb

It would be nice if the addition of `--no-total` would remove the totals from all groups and not just the main total.
The example above would output like:

```
Balance Sheet 2020-05-05

             ||         2020-05-05
=============++====================
 Assets      ||
-------------++--------------------
 Assets:A    ||           10 CURRA
 Assets:B    ||           10 CURRB
=============++====================
 Liabilities ||
-------------++--------------------
-------------++--------------------
             ||
```

This would be useful because the total, by showing all currencies in horizontal, will be real long when you have many different currencies (eg: stocks, currencies, crypto, etc).
Plus, that's what my understanding of `--no-total` means. Having no totals in any group of the report.

**Comment by Xitian9** [comment]
It seems like this is caused by explicitly setting `no_totals_` to False. Fixing this should be as simple as just removing that. Does anyone know why that choice was made in the first place?

https://github.com/simonmichael/hledger/blob/562480efeaf4b54073c75afb0b7c05035ec83451/hledger/Hledger/Cli/CompoundBalanceCommand.hs#L327-L329

**Comment by adept** [comment]
It is coming from:
```
commit 8851ebc29f877afe6756880650d323755f60a264
Author: Simon Michael <simon@joyful.com>
Date:   Tue Jul 25 19:43:45 2017 -0700

    bs/cf/is: support --output-file and --output-format=txt|csv
    
    The CSV output should be reasonably ok for dragging into
    a spreadsheet and reformatting.
```

and by the looks of it, this particular bit was not introduced as a response to a bug, so I am going to guess that this was a personal preference, maybe? @simonmichael , do you remember?

**Comment by simonmichael** [comment]
I don't remember, see how everything works without it ?

**Comment by simonmichael** [comment]
@Xitian9's #1260 is available for testing.

@makdisse wanted this as a workaround to reduce the width of multicommodity reports, while still showing the original commodities in each account. It does tend to achieve this for --flat reports (not --tree ones).

I guess I'll merge it soon if there are no objections.

**Comment by simonmichael** [comment]
@makdisse: now resolved in master by #1260.

**Comment by simonmichael** [comment]
For the record: bounty was paid (and donated to the project) along with [#1236's](https://github.com/simonmichael/hledger/issues/1236#issuecomment-648220996).
